prisma/infra/index.js

```diff
@@ -0,0 +1,145 @@
+const { PrismaClient, load_balancer_scheme_enum, task_definition_cpu_memory_enum } = require('@prisma/client')
+
+const pkg = require('./package.json');
+// TODO replace with your desired project name
+const PROJECT_NAME = pkg.name;
+
+const CONTAINER_MEM_RESERVATION = 8192; // in MiB
+const PORT = 8088;
+
+const prisma = new PrismaClient()
+
+// TODO `prisma generate` using camelcase
+// TODO use Redwood JS data migrations https://redwoodjs.com/docs/data-migrations
+// or add uniqueness constraints and use upserts to make the script idempotent
+async function main() {
+  /* AWS SECURITY GROUPS */
+  const sg = await prisma.security_group.create({
+    data: {
+      description: `${PROJECT_NAME} security group`, group_name: `${PROJECT_NAME}-security-group`,
+    },
+  });
+  // TODO replace cidr with string or prisma to support cidr so we can use entity
+  await prisma.$executeRawUnsafe(`
+    INSERT INTO security_group_rule (is_egress, ip_protocol, from_port, to_port, cidr_ipv4, description, security_group_id)
+    VALUES (false, 'tcp', ${PORT}, ${PORT}, '0.0.0.0/0', '${PROJECT_NAME}-security-group', ${sg.id});
+  `);
+  await prisma.$executeRawUnsafe(`
+    INSERT INTO security_group_rule (is_egress, ip_protocol, from_port, to_port, cidr_ipv4, description, security_group_id)
+    VALUES (true, '-1', -1, -1, '0.0.0.0/0', '${PROJECT_NAME}-security-group', ${sg.id});
+  `);
+  /*await prisma.security_group_rule.createMany({
+    data: [
+      {
+        is_egress: false,
+        ip_protocol: 'tcp',
+        from_port: PORT,
+        to_port: PORT,
+        cidr_ipv4: '0.0.0.0/0',
+        description: `${PROJECT_NAME}-security-group`,
+        security_group_id: sg.id,
+      },
+      {
+        is_egress: true,
+        ip_protocol: '-1',
+        from_port: -1,
+        to_port: -1,
+        cidr_ipv4: '0.0.0.0/0',
+        description: `${PROJECT_NAME}-security-group`,
+        security_group_id: sg.id,
+      },
+    ]
+  })*/
```

Comment:
Forget to delete this?

Reply:
I kept it there as I was adding the TODO, but will remove